glusterd2/store/liveness.go

```diff
@@ -39,19 +41,26 @@ func (s *GDStore) IsNodeAlive(nodeID interface{}) bool {
 	key := LivenessKeyPrefix + keySuffix
 	resp, err := s.Get(ctx, key)
 	if err != nil {
-		return false
+		return 0, false
 	}
 
-	return resp.Count == 1
+	if resp.Count == 1 {
+		pid := string(resp.Kvs[0].Value)
+		id, err := strconv.Atoi(pid)
+		if err != nil {
```

Comment:
You have to this error as it's very serious one. `Atoi` should not fail.

Reply:
done